test/ilpQuoterSpec.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const nock = require('nock')
+nock.enableNetConnect(['localhost'])
```

Comment:
Ok, please be careful not to write code that you don't understand. This is to allow nock to make actual socket connections. If you are not using it then I think it should be removed.